Moved docs to starter guide (#1933)
zenml-io/zenml: docs/book/user-guide/starter-guide/switch-to-production.md

Both options offer distinct advantages, allowing you to choose the deployment approach that best aligns with your organization's needs and infrastructure preferences. Whichever path you select, ZenML facilitates a seamless and efficient way to take advantage of the ZenML Server and enhance your machine learning workflows for production-level success.

<figure><img src="https://static.scarf.sh/a.png?x-pxid=f0b4f458-0a54-4fcd-aa95-d5ee424815bc" alt="ZenML Scarf"><figcaption></figcaption></figure>

## Using the ZenML CLI to connect to a deployed ZenML Server

> ZenML Cloud uses the Command Line Interface (CLI) to connect to a server. This

Reviewer: ```suggestion
[ZenML Cloud](https://cloud.zenml.io/) uses the Command Line Interface (CLI) to connect to a server. This
```

can be executed with the command:

```bash
zenml connect --url https://...
```

> This command will initiate a browser device flow. Users can choose whether to

Reviewer: ```suggestion
This command will start a series of steps to validate the device from where you are connecting that will happen in your browser. You can choose whether to
```

> mark their respective device as trusted or not. If you choose not to

Reviewer: ```suggestion
mark your respective device as trusted or not. If you choose not to
```

click `Trust this device`, a 24-hour token will be issued for authentication
services. Choosing to trust the device will issue a 30-day token instead.

To see all devices you've permitted, use the following command:

```bash
zenml authorized-device list
```

Additionally, the following command allows you to more precisely inspect one of
these devices:

```bash
zenml authorized-device describe <DEVICE_ID>  
```

For increased security, you can invalidate a token using the `zenml device lock`
command followed by the device ID. This helps provide an extra layer of security
and control over your devices.

```
zenml authorized-device lock <DEVICE_ID>  
```

To keep things simple, we can summarize the steps:

1. Use the `zenml connect --url` command to start a device flow and connect to a
   zenml server.
2. Choose whether to trust the device when prompted.
3. Check permitted devices with `zenml devices list`.
4. Invalidate a token with `zenml device lock ...`.

### Important notice

> Using the ZenML CLI is a secure and comfortable way to interact with your zenml

Reviewer: ```suggestion
Using the ZenML CLI is a secure and comfortable way to interact with your ZenML
```
